CHANGELOG.md

```diff
@@ -4,13 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.0.0-beta.2] - Unreleased
```

Comment:
this is wrong, [Unreleased] contains the changelog of master (where we reworked the interface editor)
`## [1.0.0-beta.2] - Unreleased` is where changed the login layout